web/controllers/v1/user_controller.ex

```diff
@@ -56,4 +58,29 @@ defmodule Cog.V1.UserController do
     Repo.delete!(user)
     send_resp(conn, :no_content, "")
   end
+
+  ###############################################
+  # Plug function - local only to user controller
+  ###############################################
+  @spec check_self_updating(Plug.Conn.t, []) :: Plug.Conn.t
```

Comment:
This spec is saying that the second argument is an empty list. You'd want `List.t` at least, but in this case, I think it's really `Keyword.t`